### PR TITLE
Add more self applied effects

### PR DESCRIPTION
**Pull request by TikaelSol**
Also renames a few effects missing the `Effect:` prefix and do a bit more cleanup / icon applying.

### CODE REVIEW

File: packs/pathfinder-bestiary/goblin-war-chanter.json [hunk 1]
```diff
@@ -1542,7 +1542,7 @@
...
+                    "value": "<p>The war chanter sings annoying goblin songs, distracting foes with silly and repetitive lyrics. The chanter attempts a @Check[type:performance|traits:auditory,concentrate,linguistic] check against the Will DCs of up to two enemies within @Template[type:emanation|distance:30]{30 feet}. This has the usual traits and restrictions for a Performance check.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 minute. @UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Critical Success)]{Goblin Song (Critical Success)}</p>\n<p><strong>Success</strong> As critical success, but the target is affected for only one round. @UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Success)]{Goblin Song (Success)}</p>\n<p><strong>Critical Failure</strong> The target is temporarily immune to Goblin Song for 1 hour.</p>"
```
Comment (simonward) [inline packs/pathfinder-bestiary/goblin-war-chanter.json:1545]:
```suggestion
                    "value": "<p>The war chanter sings annoying goblin songs, distracting foes with silly and repetitive lyrics. The chanter attempts a @Check[type:performance|traits:auditory,concentrate,linguistic] check against the Will DCs of up to two enemies within @Template[type:emanation|distance:30]{30 feet}. This has the usual traits and restrictions for a Performance check.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 minute. @UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Critical Success)]</p>\n<p><strong>Success</strong> As critical success, but the target is affected for only one round. @UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Success)]</p>\n<p><strong>Critical Failure</strong> The target is temporarily immune to Goblin Song for 1 hour.</p>"
```

File: packs/feats/goblin-song.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p>You sing annoying goblin songs, distracting your foes with silly and repetitive lyrics. Attempt a @Check[type:performance|defense:will] check against the Will DC of a single enemy within 30 feet. This has all the usual traits and restrictions of a Performance check. You can affect up to two targets within range if you have expert proficiency in Performance, four if you have master proficiency, and eight if you have legendary proficiency.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 minute.<br />@UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Critical Success)]{Goblin Song (Critical Success)}</p>\n<p><strong>Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 round.<br />@UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Success)]{Goblin Song (Success)}</p>\n<p><strong>Critical Failure</strong> The target is temporarily immune to attempts to use Goblin Song for 1 hour.</p>"
```
Comment (simonward) [inline packs/feats/goblin-song.json:14]:
```suggestion
            "value": "<p>You sing annoying goblin songs, distracting your foes with silly and repetitive lyrics. Attempt a @Check[type:performance|defense:will] check against the Will DC of a single enemy within 30 feet. This has all the usual traits and restrictions of a Performance check. You can affect up to two targets within range if you have expert proficiency in Performance, four if you have master proficiency, and eight if you have legendary proficiency.</p>\n<hr />\n<p><strong>Critical Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 minute.<br />@UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Critical Success)]</p>\n<p><strong>Success</strong> The target takes a -1 status penalty to Perception checks and Will saves for 1 round.<br />@UUID[Compendium.pf2e.feat-effects.Item.Effect: Goblin Song (Success)]</p>\n<p><strong>Critical Failure</strong> The target is temporarily immune to attempts to use Goblin Song for 1 hour.</p>"
```

File: packs/feat-effects/effect-weapon-tampered-with-critical-success.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
     "name": "Effect: Weapon Tampered With (Critical Success)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Tamper]</p>\n<p>Your weapon has been tampered with! You take a -2 circumstance penalty to attack rolls and damage rolls with that weapon. The effect lasts until you @UUID[Compendium.pf2e.actionspf2e.Item.Interact] to remove it.</p>"
+            "value": "<p>Granted by <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.feats-srd.Item.SheifYobjKqyK3Fv\" data-id=\"SheifYobjKqyK3Fv\" data-type=\"Item\" data-pack=\"pf2e.feats-srd\" data-tooltip=\"Feat/Feature Item\"><i class=\"fa-solid fa-medal\"></i>Tamper</a></p>\n<p>Your weapon has been tampered with! You take a -2 circumstance penalty to attack rolls and damage rolls with that weapon. The effect lasts until you <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.actionspf2e.Item.pvQ5rY2zrtPI614F\" data-id=\"pvQ5rY2zrtPI614F\" data-type=\"Item\" data-pack=\"pf2e.actionspf2e\" data-tooltip=\"Ability Item\"><i class=\"fa-solid fa-person-running-fast\"></i>Interact</a> to remove it.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-weapon-tampered-with-critical-success.json:7]: Cruft.

File: packs/feats/favored-terrain.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
...
...u aren't affected by severe or extreme heat, and you can walk along sand at full Speed without needing to Balance.</li>\n<li><strong>Forest, Mountain, or Underground</strong> You gain a climb Speed equal to your Speed. If you already had a climb Speed, you gain a +10-foot status bonus to your climb Speed.</li>\n<li><strong>Plains</strong> You gain a +10-foot status bonus to your land Speed.</li>\n<li><strong>Sky</strong> You gain a +10-foot status bonus to your fly Speed, if you have one.</li>\n<li><strong>Swamp</strong> You can move across bogs at full Speed, even if they are deep enough to be greater difficult terrain or to normally require you to Swim.</li>\n</ul>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Gain Climb Speed)]{Favored Terrain (Gain Climb Speed)}</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Increase Climb Speed)]{Favored Terrain (Increase Climb Speed)}</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Gain Swim Speed)]{Favored Terrain (Gain Swim Speed)}</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Increase Swim Speed)]{Favored Terrain (Increase Swim Speed)}</p>"
```
Comment (simonward) [inline packs/feats/favored-terrain.json:14]:
```suggestion
            "value": "<p>You have studied a specific terrain to overcome its challenges. Choose aquatic, arctic, desert, forest, mountain, plains, sky, swamp, or underground as your favored terrain.</p>\n<p>When in that terrain, you can ignore the effects of non-magical difficult terrain.</p>\n<p>If you have the @UUID[Compendium.pf2e.classfeatures.Item.Wild Stride] class feature, you gain a second benefit while in your favored terrain, depending on your choice.</p>\n<ul>\n<li><strong>Aquatic</strong> You gain a swim Speed equal to your Speed. If you already had a swim Speed, you gain a +10-foot status bonus to your swim Speed.</li>\n<li><strong>Arctic</strong> You need to eat and drink only one-tenth as much as usual, you aren't affected by severe or extreme cold, and you can walk across ice and snow at full Speed without needing to Balance.</li>\n<li><strong>Desert</strong> You need to eat and drink only one-tenth as much as usual, you aren't affected by severe or extreme heat, and you can walk along sand at full Speed without needing to Balance.</li>\n<li><strong>Forest, Mountain, or Underground</strong> You gain a climb Speed equal to your Speed. If you already had a climb Speed, you gain a +10-foot status bonus to your climb Speed.</li>\n<li><strong>Plains</strong> You gain a +10-foot status bonus to your land Speed.</li>\n<li><strong>Sky</strong> You gain a +10-foot status bonus to your fly Speed, if you have one.</li>\n<li><strong>Swamp</strong> You can move across bogs at full Speed, even if they are deep enough to be greater difficult terrain or to normally require you to Swim.</li>\n</ul>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Gain Climb Speed)]</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Increase Climb Speed)]</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Gain Swim Speed)]</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Favored Terrain (Increase Swim Speed)]</p>"
```

File: packs/feat-effects/effect-weapon-tampered-with-critical-success.json (hunk 3 above)
Comment (TikaelSol) [inline packs/feat-effects/effect-weapon-tampered-with-critical-success.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Tamper]</p>\n<p>Your weapon has been tampered with! You take a -2 circumstance penalty to attack rolls and damage rolls with that weapon. The effect lasts until you @UUID[Compendium.pf2e.actionspf2e.Item.Interact] to remove it.</p>"
```

**Pull request merged**